app/models/match_decisions/eleven/eleven_confirm_match_success_dnd_staff.rb

```diff
@@ -0,0 +1,109 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchDecisions::Eleven
+  class ElevenConfirmMatchSuccessDndStaff < ::MatchDecisions::Base
+    include MatchDecisions::AcceptsDeclineReason
+    include MatchDecisions::RouteElevenDeclineReasons
+    include MatchDecisions::RouteElevenCancelReasons
+    # validate :note_present_if_status_rejected
```

Comment:
```suggestion
```

Reply:
This has been included in the latest commit